_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
I have a suggestion: we could use a color as the main indicator of the translation relevance. The idea itself is pretty simple:
* Display a colored circle next to the lesson name (red, orange, yellow or green). It will indicate how actual the translation is.
* At the first time, likely, the user will not know what does the color mean. So the whole header should be clickable. After a click an information block should appear:
  * It will briefly describe the difference between the original lesson and the translation.
  * It may also contain a call to potential contributors.

This is quite a spontaneous idea so there are some unanswered questions:
* Displaying the number of changes will require a lot of translation work and a more complex mechanism to compose the result string. We can probably use just a static text for every case like `There are _some_ changes...`.
* Maybe the information block should always be displayed instead of appearing after some user action.
* I am still not sure where should the links head to.
* It could likely be done way better in terms of design.

Click [here](http://imgur.com/a/GvZdB) to see some screenshots of a very raw sketch.

Any thoughts?
/cc @doomspork @brain-geek @gemantzu @Koziolek @pragmaticivan @devleoper and everyone else.